utils/wasm-builder/src/wasm_project.rs

```diff
@@ -261,35 +261,93 @@ extern "C" fn metahash() {{
         Ok(())
     }
 
-    /// Post-processing after the WASM binary has been built.
-    ///
-    /// - Copy WASM binary from `OUT_DIR` to `target/wasm32-unknown-unknown/<profile>`
-    /// - Generate optimized and metadata WASM binaries from the built program
-    /// - Generate `wasm_binary.rs` source file in `OUT_DIR`
-    pub fn postprocess(&self) -> Result<()> {
-        let file_base_name = self
-            .file_base_name
-            .as_ref()
-            .expect("Run `WasmProject::generate()` first");
-
-        let from_path = self
-            .target_dir
-            .join(format!("wasm32-unknown-unknown/{}", self.profile))
-            .join(format!("{}.wasm", &file_base_name));
-
-        fs::create_dir_all(&self.target_dir)?;
-        fs::create_dir_all(&self.wasm_target_dir)?;
+    pub fn postprocess_meta(
```

Comment:
It would be great to add some docs here.

Reply:
Done